sql/scan.go

```diff
@@ -401,9 +409,9 @@ func (n *scanNode) computeOrdering(columnIDs []ColumnID) []int {
 			}
 		}
 		if !found {
```

Comment:
could you replace `found` by `len(ordering) == j` (where `j` is your `columnIDs` index)?